core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -502,7 +516,7 @@ class ContainerPool(
      * This method will introduce a stem cell container into the system.
      * If container creation fails, the container will not be entered into the pool.
      */
-    private def addWarmNodejsContainer()(implicit transid: TransactionId) = try {
+    private def addWarmNodejsContainer()(implicit transid: TransactionId) = Future { Try {
```

Comment:
`Future[Try[...]]` really necessary? `Future[...]` should be enough (already is a `Try`)